layouts/calendar_layout.py

```diff
@@ -0,0 +1,81 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import layouts.style.style as style
+import calendar
+from tools.calendar_generation import generate_month
+from tools.calendar_generation import print_title
+
+CAL_BUT = [
+        [
+           'Previous'
+	],
+        [
+           'Next'
+        ]
+]
+
+def change_calendar_content():
+   """
+   Create layout for calendar
+   
+   :return: Layout with 1 second refresh interval
+   :rtype: dash.development.base_component.ComponentMeta
+   """
+   return html.Div([
+        html.Div(id='live-calendar'),
+        html.Div(id='calendar-output'),
+        dcc.Interval(
+            id='calendar-interval-component', interval=1000, n_intervals=0)
+        ], style={'margin': style.PADDING})
+
+def create_all_calendar():
+   """
+   Create all content for a calendar (one month)
+   
+   :return: Output from create_calendar_content function and title of a calendar
+   :rtype: dash.development.base_component.ComponentMeta
+   """
+   return html.Div(
+      [
+         html.H1('Desk reservations'),
+         html.H2(print_title()),
+         create_calendar_content()
+      ])
+
+def create_calendar_content():
+    """
+    Create buttons for a calendar - with days and switching months
+    
+    :return: Buttons with days of the month and switching months buttons
+    :rtype: dash.development.base_component.ComponentMeta
+    """
+    calendar_buttons = []
+    new_calendar = generate_month()
+    change_month_layout = []
+    for calbuttons in CAL_BUT:
+         for key in calbuttons:
```

Comment:
As a result of nested lists you have N^2 operations here.
I think that using just `chidren=[callbutton]` with one for loop would be optimal

Reply:
I didn't get it. I understand that I should simplify the code, but I can't get a way how to do it as you described.

Reply:
Ok, I had to think for a while, now I think I know what you meant.

Reply:
You create nested lists with just one string
`[[‘Previous’], [‘Next’]]`
And then you do for on main list and then for on sublists with just one string inside.

You can make simple list [‘Previous’, ‘Next’] and then create only one for loop.
and in field children just pass `[variablewithstring]`.